fastlane/lib/fastlane/actions/appetize_url_generator.rb

```diff
@@ -0,0 +1,107 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class AppetizeUrlGeneratorAction < Action
+      def self.run(params)
+        link = "https://appetize.io/embed/#{params[:public_key]}"
+
+        url_params = []
+        url_params << "autoplay=true"
+        url_params << "orientation=#{params[:orientation]}"
+        url_params << "device=#{params[:device]}"
+        url_params << "deviceColor=#{params[:color]}"
+        url_params << "scale=#{params[:scale]}"
+        url_params << "launchUrl=#{params[:launch_url]}" if params[:launch_url]
+        url_params << "language=#{params[:language]}" if params[:language]
+
+        return link + "?" + url_params.join("&")
+      end
+
+      #####################################################
+      # @!group Documentation
+      #####################################################
+
+      def self.description
+        "Generate an URL for appetize simulator"
+      end
+
+      def self.details
+      end
+
+      def self.available_options
+        [
+          FastlaneCore::ConfigItem.new(key: :public_key,
+                                       env_name: "APPETIZE_PUBLICKEY",
+                                       description: "Public key of the app you wish to update",
+                                       is_string: true,
```

Comment:
FYI `is_string` isn't fully deprecated, because of the cases of Boolean but I submitted 2 alternative solutions to the problem today, like #4142. But as @mfurtak  said, for Strings, yes, you can already use `type`.